magma/passes/clock.py

```diff
@@ -1,5 +1,5 @@
 from ..is_definition import isdefinition
-from ..clock import wiredefaultclock
+from ..clock import wiredefaultclock, wireclock
```

Comment:
also here? related?

Reply:
This was necessary for a test

Reply:
The backends automatically wire up clocks and clock related signals (reset, enable, ...).  For the simulator, we need to run these passes independently since they don't use a backend

Reply:
got it. in that case, should this be called directly by the simulator codepath? rather than adding it to the general pass. fine by me if we think it's generally useful to have this, and should be in the general pass

Reply:
I think it's generally useful, we can have the simulator call it by default too though